config/admins.txt

```diff
@@ -139,3 +139,4 @@ Denton-30 = Game Master
 Naksuasdf = Game Master
 MrDoomBringer = Game Master
 shizcalev = Game Master
+NicBR = Game Master

```

Comment:
When I saw that there was the Config Option tag, I was about to shriek that we didn't need being able to click on people under plastic flaps as a config option.

But this is fine.

Reply:
lol

Reply:
nice